Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -160,7 +109,7 @@ else
 	
         Write-Output "Copy will be Started for - $machine"
 
-        $job = Start-Job -ScriptBlock $CopyJob -ArgumentList $environmentName, $envOperationId, $machine, $sourcePath, $targetPath, $machineUserName, $machinePassword, $cleanTargetBeforeCopy
+        $job = Start-Job -ScriptBlock $CopyJob -ArgumentList $environmentName, $envOperationId, $machine, $sourcePath, $targetPath, $machineUserName, $machinePassword, $cleanTargetBeforeCopy, $connection
 
         $Jobs.Add($job.Id, $machine)
     
```

Comment:
we are making two calls to Get-Job in the while loop and then immediately in the foreach loop after 10 seconds. Can we look to see how to optimize that?